api/v1beta1/verticadb_webhook.go

```diff
@@ -307,6 +338,38 @@ func (v *VerticaDB) validateCommunalPath(allErrs field.ErrorList) field.ErrorLis
 	return append(allErrs, err)
 }
 
+func (v *VerticaDB) validateS3ServerSideEncryption(allErrs field.ErrorList) field.ErrorList {
```

Comment:
A few other things to validate:
- should we validate if someone sets the `communal.s3SseCustomerKeySecret` without setting `communal.s3ServerSideEncryption`?
- if `communal.s3ServerSideEncryption` is set but the communal path isn't s3?